Added tool description for CPA-BAM-Slicing

schrodibear opened the pull request:
The only difference from basic CPAchecker is executing the wrapper rather than cpa.sh

PhilippWendler commented:
If the only difference to `cpachecker.py` is the use of `wrapper.sh`, then this should be refactored such that all duplicate code is avoided.

Btw., are you sure that you want to call the main entry point of your tool `wrapper.sh` and not something more descriptive?
If your tool is based on CPAchecker, you could also integrate your extensions into for example `cpa.sh` itself such that no new tool-info module is needed.

schrodibear commented:
I moved slicer invocation to cpa.sh in the submission and changed tool info in the benchmark definition to just "cpachecker" ([here](https://github.com/sosy-lab/sv-comp/pull/51)). So this PR can be closed.
